src/applications/personalization/dashboard/isPersonalizationEnabled.js

```diff
@@ -1,6 +1,3 @@
-import dashboardManifest from './manifest';
-
 export default function isPersonalizationEnabled() {
-  if (__BUILDTYPE__ !== 'production') return true;
-  return dashboardManifest.production;
+  return true;
 }
```

Comment:
This should ultimately be torn down entirely, but that's outside the scope of this PR.